src/apps/service/apps/jobs/submissions/async.clj

```diff
@@ -56,73 +55,73 @@
         job-config (substitute-param-values path-map (:job_config submission config))
         config     (substitute-param-values path-map config)]
     (assoc submission :job_config job-config
```

Comment:
This `:job_config job-config` k-v pair could also be moved to a new line for consistency.